api/internal/loader/fileloader_test.go

```diff
@@ -659,6 +660,35 @@ func TestLoaderHTTP(t *testing.T) {
 		_, err := l2.Load(x.path)
 		require.Error(err)
 	}
+
+	var testCaseRedirect = []testData{
+		{
+			path:            "https://example.com/resource.yaml",
+			expectedContent: "https content",
+		},
```

Comment:
Looking at the code below, do you plan to add more tests?
https://github.com/kubernetes-sigs/kustomize/pull/5678/files#diff-e89184db286ae331249443264748758337bf37c3e9c92a32d6102f94aab422f9R686-R690

Reply:
I have no plans to add another tests.